app/src/main/res/values/strings.xml

```diff
@@ -864,7 +864,7 @@
     <!-- History overflow menu open in private tab button -->
     <string name="history_menu_open_in_private_tab_button" moz:removedIn="94" tools:ignore="UnusedResources">Open in private tab</string>
     <!-- Text for the button to delete a single history item -->
```

Comment:
IIRC this used to be for the actual button text. Let's update this to specify this is a content description string now.

```suggestion
    <!-- Context description text for the button to delete a single history item -->
```